### ISSUE TITLE
CVXNet Training with custom data.

### ISSUE THREAD

**Issue by yeshwanth95** [tensorflow/graphics#683]
Hi @Awcrr 
I'm trying to reimplement the cvxnet model in pytorch and train for a custom dataset of images. I intend to do single view reconstruction but for the 2D case i.e output 2D polygons/shapefiles instead of 3D meshes. I have a custom dataset of images and corresponding gt raster seg masks. Can you please tell me how to preprocess this data for training with cvxnet?

In particular, I'm confused about how to generate the sample points and the corresponding point labels for this dataset.
If I have to guess, I sample random coordinates from the image array (x, y) and get their corresponding label from the raster gt seg masks. Is that correct?

Any help will be appreciated. Looking forward to your reply.
